packages/ui-components/src/components/form/select/Badges.stories.ts

```diff
@@ -0,0 +1,62 @@
+import { Meta, StoryObj } from '@storybook/vue3'
+import Badges from './Badges.vue'
+
+type StoryType = StoryObj<
+  Record<string, unknown> & {
+    'update:modelValue': (val: unknown) => void
+  }
+>
+
+type SingleItem = {
+  id: string
+  text: string
+}
+
+export default {
+  component: Badges,
+  argTypes: {
+    'update:modelValue': {
+      action: 'update:modelValue',
+      type: 'function'
+    }
+  }
+} as Meta
+
+export const Default: StoryType = {
+  render: (args, ctx) => ({
+    components: { Badges },
+    setup: () => {
+      return { args }
+    },
+    template: `
+    <div class="flex justify-center h-72 w-full">
+      <Badges v-bind="args" @update:modelValue="onModelUpdate" class="w-full"/>
+    </div>
+    `,
+    methods: {
+      onModelUpdate(val: Array<SingleItem>) {
```

Comment:
This is what I meant in the other comment in Switch.stories.ts: It's kind of verbose and hacky, but unfortunately its the best way Storybook offers us to properly have bidirectional binding for Storybook args.

Reply:
https://github.com/specklesystems/speckle-server/pull/1792/commits/0c37aa146691e9d5b46eac68742c005ac6ed1c64